### DISCUSSION TITLE
Is the update check already deprecated for TryBuild?

### DISCUSSION THREAD

**Discussion by cary-rowen** [nvaccess/nvda#14493] [Q&A] [answered]
Just curious.
When I tested #14484, I found that this version of the build cannot find the option to check new versions under the help menu.

**Comment by seanbudd** [comment, marked as answer]
You cannot update from a try build. Same as when running from source.

**Comment by Brian1Gaff** [comment]
I don't think you ever could, at least most of the try builds I tested when 
issues were being fixed never did have the function. Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----
